buildenv/jenkins/common/variables-functions.groovy

```diff
@@ -771,6 +771,27 @@ def add_pr_to_description() {
     }
 }
 
+def setup() {
+    set_job_variables(params.JOB_TYPE)
+
+    switch (params.JOB_TYPE) {
+        case "pipeline":
+            buildFile = load 'buildenv/jenkins/common/pipeline-functions.groovy'
```

Comment:
`pipeline-functions.groovy` is loaded twice: here as `buildFile` and at L203:` pipelineFunctions = load 'buildenv/jenkins/common/pipeline-functions.groovy'`
Could use: `buildFile = pipelineFunctions `

Reply:
Changed